mesheryctl/internal/cli/root/pattern/apply_test.go

```diff
@@ -72,6 +72,34 @@ func TestApplyCmd(t *testing.T) {
 			Token:       filepath.Join(fixturesDir, "token.golden"),
 			ExpectError: false,
 		},
+
+		{
+			Name:             "Apply Pattern from URL",
+			Args:             []string{"apply", "-f", "https://raw.githubusercontent.com/yash37158/SamplePattern/main/samplepatternwithservicemesh.golden"},
+			ExpectedResponse: "apply.url.output.golden",
+			URLs: []utils.MockURL{
+				{
+					Method:       "GET",
+					URL:          "https://raw.githubusercontent.com/yash37158/SamplePattern/main/samplepatternwithservicemesh.golden",
```

Comment:
I think it is best to keep test related assets within this repo itself and not have dependency on files from external repositories. 
We have a similar design file in the `fixtures` directory [here](https://github.com/meshery/meshery/blob/master/mesheryctl/internal/cli/root/pattern/fixtures/pattern.golden). 
We can use a link to that instead (https://raw.githubusercontent.com/meshery/meshery/master/mesheryctl/internal/cli/root/pattern/fixtures/pattern.golden)

Reply:
Yes sure @DevRish